libr/anal/fcn.c

```diff
@@ -973,6 +1022,8 @@ static int fcn_recurse(RAnal *anal, RAnalFunction *fcn, ut64 addr, ut8 *buf, ut6
 		r_list_free (list);
 	}
 	ut64 movptr = UT64_MAX; // used by jmptbl when coded as "mov reg,[R*4+B]"
+	int len = anal->opt.bb_max_size;
+	ut8 buf[32]; // 32 bytes is enough to hold any instruction.
```

Comment:
Not strictly true. Dalvik's packed-switch-payload pseudo-instruction can be huge https://source.android.com/devices/tech/dalvik/dalvik-bytecode#packed-switch

Reply:
in rasm this is handled with RBuffer, so the instruction can hold any amount of data without problems, we should probably have tests to see if this broke anything, but it was 128 bytes before, and dalvik arrays and packed switches can be MUCH bigger than 128 bytes

Reply:
Yes but `r_anal_op` wasn't limited to 32 bytes before (https://github.com/radare/radare2/pull/13223/files#diff-49a52329a1b3f5e5f62d571577391074L1045) ... tests are really needed here